README.md

```diff
@@ -94,7 +94,9 @@ Fill in the missing pieces in [this](https://github.com/Azure/kubernetes-keyvaul
 
     |Name|Required|Description|Default Value|
     |---|---|---|---|
-    |usepodidentity|no|specify access mode: service principal or pod identity|"false"|
+    |usepodidentity|no|specify access mode: service principal or pod identity or azure managed identity|"false"|
+    |usemanagedidentity|no|specify access mode: use service principal or pod identity or azure managed identity|"false"|
+    |managedidentityclientid|no|specify the azure managed identity client id to use. specify empty to use the system assigned identity on VMSS|""|
```

Comment:
can we rename this to `vmuserassignedidentityclientid` everywhere?

Reply:
in the description, can we reword this: `When using VM managed identity, specify the VM user assigned identity client id.  If empty, then defaults to use the system assigned identity on the VM.`

Reply:
`vmuserassignedidentityclientid` will be a bit too long? how about `vmmanagedidentityclientid`? so we keep the `vmmanagedidentity` term everywhere.